drivers/regulator/regulator_axp192.c

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright (c) 2021 NXP
+ * Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT x_powers_axp192_regulator
+
+#include <errno.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/regulator.h>
+#include <zephyr/sys/linear_range.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/dt-bindings/regulator/axp192.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/logging/log_instance.h>
+
+LOG_MODULE_REGISTER(regulator_axp192, CONFIG_REGULATOR_LOG_LEVEL);
+
+/* Output control registers */
+#define AXP192_REG_EXTEN_DCDC2_CONTROL   (0x10u)
+#define AXP192_REG_DCDC123_LDO23_CONTROL (0x12u)
+#define AXP192_REG_DCDC2_VOLTAGE         (0x23u)
+#define AXP192_REG_DCDC2_SLOPE           (0x25u)
+#define AXP192_REG_DCDC1_VOLTAGE         (0x26u)
+#define AXP192_REG_DCDC3_VOLTAGE         (0x27u)
+#define AXP192_REG_LDO23_VOLTAGE         (0x28u)
+#define AXP192_REG_DCDC123_WORKMODE      (0x80u)
```

Comment:
```suggestion
#define AXP192_REG_EXTEN_DCDC2_CONTROL   0x10U
#define AXP192_REG_DCDC123_LDO23_CONTROL 0x12U
#define AXP192_REG_DCDC2_VOLTAGE         0x23U
#define AXP192_REG_DCDC2_SLOPE           0x25U
#define AXP192_REG_DCDC1_VOLTAGE         0x26U
#define AXP192_REG_DCDC3_VOLTAGE         0x27U
#define AXP192_REG_LDO23_VOLTAGE         0x28U
#define AXP192_REG_DCDC123_WORKMODE      0x80U
```

Reply:
fixed in [7eb274f](https://github.com/zephyrproject-rtos/zephyr/pull/58809/commits/7eb274f10360e4f6ddb1d2c6e091cb76120844a3)

I reviewed and corrected all constants in mfd and regulator driver.